deploy/crs/v2/snippets/common_config.yaml

```diff
@@ -11,4 +11,5 @@ spec:
   commonConfig:
     # By default is the cr.Name
     applicationName: other
-    adminPassword: example
+    secretAdminCredentialsReference:
+      name: "kie-admin-credentials"
```

Comment:
let's create a new snippet with example of usage of user credentials secret

Reply:
ok